test/over_react_test/zone_render_tests.dart

```diff
@@ -0,0 +1,131 @@
+import 'dart:async';
+
+import 'package:over_react_test/over_react_test.dart';
+import 'package:react/react_client/react_interop.dart';
+import 'package:test/test.dart';
+
+import 'custom_matchers_test.dart';
+import 'helper_components/sample_component.dart';
+
+main() {
+  group('TestJacket mount', () {
+    sharedZoneRenderTests(mount);
+  });
+
+  group('render', () {
+    sharedZoneRenderTests(render);
+  });
+
+  group('renderAttachedToDocument', () {
+    sharedZoneRenderTests(renderAttachedToDocument);
+  });
+}
+
+void sharedZoneRenderTests(Function(ReactElement element, {bool autoTearDown}) renderFunction) {
+  group('sharedZoneRenderTests:', () {
+    setUp(() {
+      setComponentZone(Zone.root);
+    });
+
+    test('Failing expects work in Component2 lifecycle methods', () {
```

Comment:
expects work?

Reply:
Haha yeahh that wording is pretty gross, huh? I updated it to read better!